Python/optimizer_bytecodes.c

```diff
@@ -396,7 +396,6 @@ dummy_func(void) {
     op(_TO_BOOL_LIST, (value -- res)) {
         int already_bool = optimize_to_bool(this_instr, ctx, value, &res);
         if (!already_bool) {
-            sym_set_type(value, &PyList_Type);
```

Comment:
Does CONTAIN_OP_SET and CONTAIN_OP_DICT need to remove their sym setting type as well? Or is that not even happening at the moment?

Reply:
Not happening currently,. `_CONTAINS_OP_SET` can't even set anything, since it handles both `set` and `frozenset` (this might be worth re-evaluating, or splitting up).